snazzer

```diff
@@ -71,21 +71,30 @@ assert_btrfs_tools() {
     fi
 }
 
+# function duplicated in snazzer-measure
 glob2grep_file() {
     FILE=$1
     OUT=$(mktemp)
 
+    # check if every line starts with either / or *, if not it's a syntax error.
+    while read -r line; do
+        if ! [[ "$line" =~ ^[/\*].* ]]; then
```

Comment:
This is failing because [[ =~ is a bash-ism, but we are a #/bin/sh script ;-)

Perhaps try something like

```
if ! grep '^[/\*]' "$1"; then
    echo "SYNTAX ERROR: $1 contains lines that start with neither / nor *." >&2
    exit 12
fi
```